src/pocketmine/plugin/ScriptPluginLoader.php

```diff
@@ -105,7 +105,7 @@ public function getPluginDescription($file){
 				$data[$key] = $content;
 			}
 
-			if($insideHeader and strpos($line, "**/") !== false){
+			if($insideHeader and (strpos($line, "*/") !== false or strpos($line, "**/") !== false)){
```

Comment:
the `strpos($line, "**/")` is unnecessary. Any string that contains `**/` also contains `*/`, unless some black magic is at work.